Doc/whatsnew/3.0.rst

```diff
@@ -269,7 +269,7 @@ changed.
   separate *mutable* type to hold buffered binary data,
   :class:`bytearray`.  Nearly all APIs that accept :class:`bytes` also
   accept :class:`bytearray`.  The mutable API is based on
-  :class:`collections.MutableSequence`.
+  :class:`collections.abc.MutableSequence`.
```

Comment:
As a historical document, let's leave the link text reading `collections.MutableSequence`, as it was at the time, but we can let it link to the new `collections.abc.MutableSequence`.

We can do it with markup like ``` :role:`custom text <target>` ```

https://devguide.python.org/documentation/markup/